pkg/api/types.go

```diff
@@ -451,6 +451,8 @@ type ISCSIVolumeSource struct {
 	IQN string `json:"iqn,omitempty"`
 	// Required: iSCSI target lun number
 	Lun int `json:"lun,omitempty"`
+	// Optional: Defaults to 'default' (tcp). iSCSI interface name that uses an iSCSI transport.
+	ISCSIInterface string `json:"iscsiInterface,omitempty"`
```

Comment:
This covered multiple times in previous discussions, please refer previous discussions on this commit as to why this naming convention was chosen.